public/pages/DetectorResults/containers/AnomalyResults.tsx

```diff
@@ -273,6 +279,14 @@ export function AnomalyResults(props: AnomalyResultsProps) {
     }
   };
 
+  const getInitProgressMessage = () => {
+    return detector && isDetectorInitializing && detector.initProgress
+      ? `The detector needs to capture approximately
+                  ${detector.initProgress.neededShingles} data points for initializing. If your data stream is continuous, this process will take around
```

Comment:
How about change `${detector.initProgress.neededShingles} data points for initializing` to `${detector.initProgress.neededShingles} more data points for initializing`

Reply:
Sorry, seems I misunderstood it. Here the `${detector.initProgress.neededShingles} ` is the total data points needed or we still need such more data points?

Reply:
“data points” should be replaced by shingles.

Reply:
re Yaliang: we still need such more data points.

Reply:
> “data points” should be replaced by shingles.

Should we expose `shingle`, which is one internal term, to customer? I think `data points` expresses the same thing, and it is more user-friendly.

Reply:
We should expose it.  We mentioned it in our doc: https://opendistro.github.io/for-elasticsearch-docs/docs/ad/ 
 Data points and shingle are not the same thing.

Reply:
> How about change `${detector.initProgress.neededShingles} data points for initializing` to `${detector.initProgress.neededShingles} more data points for initializing`?
> 
> Confirm with tech writer about the wording.

The message is already reviewed with tech writer. I will check with him on adding the word "more".

Reply:
> We should expose it. We mentioned it in our doc: https://opendistro.github.io/for-elasticsearch-docs/docs/ad/
> Data points and shingle are not the same thing.

From the doc:
```
The anomaly detection model uses a shingling process, a technique that uses consecutive data points to create a sample for the model. This process needs a certain number of aggregated data points from contiguous intervals.
```
In my understanding, `shingles` are the data points we need to collect for shingling process. Is there any reason why we should not use `data points` here?

Reply:
hmm, I see, sound like shingles are "consecutive data points", is it right?

Reply:
A shingle is a consecutive sequence of the most recent records.

Reply:
what about using "consecutive data points", my point is that shingle is not intuitive word for user, and we only have  definition of what `shingle process` means in the [doc](https://opendistro.github.io/for-elasticsearch-docs/docs/ad/)

Reply:
after discussing offline with Kaituo, we may remove the message regarding shingle/data point which may be confusing to user, while provide needed time and percentage only.

Reply:
Wording has been confirmed with tech writer.